tests/app/db.py

```diff
@@ -541,10 +541,11 @@ def create_domain(domain, organisation_id):
     return domain
 
 
-def create_organisation(name='test_org_1', active=True):
+def create_organisation(name='test_org_1', active=True, type=None):
     data = {
         'name': name,
-        'active': active
+        'active': active,
+        'organisation_type': type
```

Comment:
It’s best not to use `type` as a variable name because it’s also the name of [a built in function](https://docs.python.org/3/library/functions.html#type).

Reply:
good point, I changed that now 👍